internal/docs/design.md

```diff
@@ -184,6 +187,103 @@ needed). However, this would result in inconsistent names over time (e.g., some
 packages would expose `New` with an `Options`, while others would expose
 `NewWithOptions`).
 
+### URLs
+
+To enable the [Backing services factor][] of a Twelve-Factor Application, Go
```

Comment:
Does the Go CDK have a policy of adhering to Twelve-Factor or some subset of it?